data/data-pipeline/data_pipeline/comparison_tool/src/utils.py

```diff
@@ -0,0 +1,330 @@
+import pandas as pd
+from data_pipeline.score import field_names
+
+
+FIPS_MAP = {
```

Comment:
Not to over-engineer things, but might be useful to use the same method/map we use for states fips codes elsewhere: `data_pipeline.etl.sources.census.etl_utils.get_state_information`. 

E.g., if there was ever a policy decision to include Palau etc, I could see us forgetting to update both separate lists.

Reply:
i totally forgot we already had this! 😂